### PR TITLE
Change pfSimpleFilter directive to use two sub-component directives

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
pfSimpleFilter directive needs to be split up so that the components for the fields area, and the components for the current results and active filters area can be placed separately on the toolbar or where ever an application would like them. Currently, this one directive takes up the entire toolbar area and won't allow (easily) for additional toolbar components.

### CODE REVIEW

File: src/filters/simple-filter-fields.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            </li>
+          </ul>
+        </div>
+        <input class="form-control" type="{{currentField.filterType}}" ng-model="config.currentValue"
+               placeholder="{{currentField.placeholder}}"
+               ng-keypress="onValueKeyPress($event)"
+               ng-if="currentField.filterType !== 'select'"/>
+        <select pf-select class="form-control filter-select" id="currentValue"
+                ng-model="config.currentValue"
+                ng-options="o as o for o in currentField.filterValues"
```
Comment (waldenraines) [inline src/filters/simple-filter-fields.html]: Is the "o as o" part needed?  I would have thought the `for o in currentField.filterValues` was sufficient.

Comment (jeff-phillips-18) [inline src/filters/simple-filter-fields.html]: It needs to have the select and the value but not the as. I agree this would be more readable if simply o was not used. I will update.

File: src/views/toolbar/data-toolbar-directive.js [hunk 2]
```diff
@@ -231,7 +231,33 @@ angular.module('patternfly.views').directive('pfDataToolbar',
         $scope.checkViewDisabled = function (view) {
           return $scope.config.viewsConfig.checkViewDisabled && $scope.config.viewsConfig.checkViewDisabled(view);
         };
+
+        $scope.filterExists = function (filter) {
+          var found = false;
+          $scope.config.filterConfig.appliedFilters.forEach(function (nextFilter) {
```
Comment (waldenraines) [inline src/views/toolbar/data-toolbar-directive.js]: Seems like we could use [`_.where`](https://lodash.com/docs#where) here to simplify this.

Comment (jeff-phillips-18) [inline src/views/toolbar/data-toolbar-directive.js]: Yep. Thanks!

File: grunt-ngdocs-index.tmpl [hunk 3]
```diff
@@ -31,7 +31,7 @@
       }
 
       addTag('base', {href: baseUrl});
-      addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
+      // addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
```
Comment (waldenraines) [inline grunt-ngdocs-index.tmpl]: Is this change intentional?

Comment (jeff-phillips-18) [inline grunt-ngdocs-index.tmpl]: Yes, sorry I squashed a commit that explained it. The css/prettify.css changes some sizing causing the filter components to not line up as they would if just using patternfly.

Comment (waldenraines) [inline grunt-ngdocs-index.tmpl]: Should we just remove this instead of commenting it out then?

Comment (jeff-phillips-18) [inline grunt-ngdocs-index.tmpl]: I left it incase we noticed that parts of the page (not patternfly components) didn't look right, we would have the hint as to where settings might be.

Comment (waldenraines) [inline grunt-ngdocs-index.tmpl]: In that case I think we should add a comment above as to why it is commented out and still here.

Comment (jeff-phillips-18) [inline grunt-ngdocs-index.tmpl]: done

**Comment by dtaylor113** [comment]
LGTM
